### PR TITLE
moved repositories to core

### PR DESCRIPTION
**Pull request by julianseeger**
depends on Qabel/qabel-core#521

### CODE REVIEW

File: src/main/java/de/qabel/desktop/daemon/sync/worker/DefaultSyncer.java [hunk 1]
```diff
@@ -160,7 +159,7 @@ private void download(ChangeEvent event) {
         }
 
         SyncIndexEntry entry = index.get(BoxFileSystem.getRoot().relativize(download.getSource()));
-        long size = download.isDir() ? DirectoryMetadata.DEFAULT_SIZE : download.getSize();
+        long size = download.isDir() ? 56320L : download.getSize();
```
Comment (audax) [inline src/main/java/de/qabel/desktop/daemon/sync/worker/DefaultSyncer.java]: A constant wouldn't be that bad…

File: src/main/java/de/qabel/desktop/storage/cache/CachedBoxNavigation.java [hunk 2]
```diff
@@ -36,12 +36,12 @@ public CachedBoxNavigation(T nav, Path path) {
...
 
     @Override
-    public DirectoryMetadata reloadMetadata() throws QblStorageException {
+    public JdbcDirectoryMetadata reloadMetadata() throws QblStorageException {
         return nav.reloadMetadata();
     }
 
     @Override
-    public void setMetadata(DirectoryMetadata dm) {
+    public void setMetadata(JdbcDirectoryMetadata dm) {
```
Comment (audax) [inline src/main/java/de/qabel/desktop/storage/cache/CachedBoxNavigation.java]: Can this be an interface?

File: src/main/resources/icon/edit_weiss.svg [hunk 3]
```diff
@@ -2,20 +2,19 @@
...
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+    width="11.288889mm"
+    height="11.288889mm"
+    viewBox="0 0 40 40"
+    id="svg21868"
+    version="1.1"
+    inkscape:version="0.91 r13725"
+    sodipodi:docname="edit_weiss.svg">
```
Comment (audax) [inline src/main/resources/icon/edit_weiss.svg]: Why is all the xml stuff reformatted?

File: src/test/java/de/qabel/desktop/ui/remotefs/FolderTreeItemTest.java [hunk 4]
```diff
@@ -132,12 +132,12 @@ private BoxFile createSomeFile() {
         public List<BoxFolder> folders = new LinkedList<>();
 
         @Override
-        public DirectoryMetadata reloadMetadata() throws QblStorageException {
+        public JdbcDirectoryMetadata reloadMetadata() throws QblStorageException {
```
Comment (audax) [inline src/test/java/de/qabel/desktop/ui/remotefs/FolderTreeItemTest.java]: Again: Can this use the interface?

File: src/main/java/de/qabel/desktop/storage/cache/CachedBoxNavigation.java (hunk 2 above)
Comment (julianseeger) [inline src/main/java/de/qabel/desktop/storage/cache/CachedBoxNavigation.java]: Not yet unfortunately. The handling of global properties like the temp dir is not there ... we should use more Factories ;)
Currently, the JdbDirectoryMetadata is for example used to extract the location of the database file and the temp dir ... to access the temp dir. We need to fix that first before we can completely use the DirectoryMetadata Interface (the change is caused by a rename + extract interface in the core)

File: src/main/resources/icon/edit_weiss.svg (hunk 3 above)
Comment (julianseeger) [inline src/main/resources/icon/edit_weiss.svg]: absolutely no idea o.O

File: src/test/java/de/qabel/desktop/ui/remotefs/FolderTreeItemTest.java (hunk 4 above)
Comment (julianseeger) [inline src/test/java/de/qabel/desktop/ui/remotefs/FolderTreeItemTest.java]: not yet, it never used an interface

**Pull request merged**